src/store/mutations.js

```diff
@@ -74,6 +74,7 @@ export default {
       {
         ...payload,
         wallet: true,
+        type: 'wallet',
```

Comment:
```suggestion
        wallet: true,
        type: 'wallet',
```
Why do we need two properties to verify is notification from wallet or not?

Reply:
@CedrikNikita @maxkpower

Yes, not visible difference between **read** and **unread** notifications because it's not implemented according to design provided. I think on the mockup below the "unread" are clearly visible.
And yes, the **...** menu icon should be right aligned with the date and the padding should be always 16px to the right. Sync with me if you'd like to have the source file where styling and measures can be taken from.

![Group 1591](https://user-images.githubusercontent.com/12225410/101617157-f2ad4580-3a18-11eb-9b11-c5956cdf80c7.png)